storybook/src/data-display/DataTable.stories.tsx

```diff
@@ -796,20 +796,30 @@ export const WithHorizontalScrollAndLastColumnFixed = () => (
   </DataTable>
 );
 
-export const WithDefaultAscendingSortByName = () => (
-  <DataTable
-    data={smallData}
-    defaultSortBy={[
+export const WithDefaultAscendingSortByName = () => {
+  const [dataTableState, dataTableHook] = useDataTable({
+    defaultSortBy: [
       {
         column: "name",
         sortDirection: "ASCENDING",
       },
-    ]}
-  >
-    <DataTable.Column header="ID" accessor="id" canSort={false} />
-    <DataTable.Column header="Name" accessor="name" canSort={true} />
-  </DataTable>
-);
+    ],
+  });
+
+  const sortedData = React.useMemo(() => {
+    return [...smallData].sort((a, b) => {
```

Comment:
can we include smallData in the story. I will open a new issue, but the general problem with stories si that we optimized them for our code, not for display, and there are missing information in a lot of code snippets in storybook.

Reply:
i have opened it https://github.com/croz-ltd/tiller/issues/176